### DISCUSSION TITLE
Word counts in article meta

### DISCUSSION THREAD

**Discussion by wizardbyron** [jpanther/congo#52] [Ideas]
Hi, I've implemented the feature that show word count in article meta (between date and reading time) on my blog post. 
You can see that in [my site](https://www.guyu.me/) (Chinese).

The implementation has:
1. A `word-count.html` under meta folder along with `reading-time.html`.
2. A feature toggle `showWordCount` in `params.article`.
3. word-count-units attributes in i18n (Only for 'en' and 'zh').

May I create PR for this feature ?

**Comment by jpanther** [comment]
Hi @wizardbyron, this sounds like a great addition. Feel free to submit a PR.

**Reply by wizardbyron** [reply]
I've created https://github.com/jpanther/congo/pull/57, please have a review.
I need some help on other languages in i18n.
